### PR TITLE
VGG 16 - 70% top-1 accuracy 

### PR DESCRIPTION
**Pull request by hasibzunair**
Repository contains codes for the initial classification algorithm(NOT pre-trained models) trained on both **Imagenet** and **Imagewoof**. Both classifiers were trained on 160px and 40 epochs max, as required. Will be pushing further progress made in the link to repository shared.  Requesting a review. 

Thanks.

### CODE REVIEW

**Comment by jph00** [comment]
You'll need to do a bit better to get in to the leaderboard - sorry! :)

**Comment by hasibzunair** [comment]
@jph00  no problem. Will work on it!
